.github/workflows/test.yaml

```diff
@@ -36,15 +40,21 @@ jobs:
                     - php-version: '8.0'
                       dependencies: highest
                       allowed-to-fail: false
-                      variant: 'symfony/symfony:"4.4.*"'
+                      variant: 'sonata-project/block-bundle:"3.*"'
```

Comment:
This is not correct afaik, sonata admin is only compatible with block 4 right?

Reply:
Indeed, fixed